messages/culture.txt

```diff
@@ -31,4 +31,17 @@
-<b>Honest retrospectives</b> Please make an effort to fill out your retrospective well. Good, honest, open and thoughtful retrospectives serve as a culture reinforcement tool. Good retrospectives help normalize the fact that all of us are struggling and doing good at the same time. This will help the new hire identify with us quicker. Open retrospectives remove the stigma around failure. The more honestly we share our experience, the more likely that the new folks will trust us. The more thoughtful our retrospectives, the more likely that our new hires will be more introspective.
\ No newline at end of file
+<b>Honest retrospectives</b> Please make an effort to fill out your retrospective well. Good, honest, open and thoughtful retrospectives serve as a culture reinforcement tool. Good retrospectives help normalize the fact that all of us are struggling and doing good at the same time. This will help the new hire identify with us quicker. Open retrospectives remove the stigma around failure. The more honestly we share our experience, the more likely that the new folks will trust us. The more thoughtful our retrospectives, the more likely that our new hires will be more introspective.
+<b>Everyone has the authority to give feedback</b> Please be observant for bad (or inefficient) habits in your colleagues. When you notice something wrong or sub-optimal with the behaviour of a colleague - speak up! You <i>do not</i> have to be their boss to speak up. As an early employee, it is in your interest to get into the habit of giving feedback. Some simple guidelines around giving feedback - give feedback to your colleagues as early as possible instead of waiting for an ideal moment. Try your best not to be too accusatory during the feedback. Asking a thought-provoking question or pointing a missing piece are also valid forms of feedback. If your feedback is received (or given) poorly, rest assured that there are several other actors in the system that will try to correct the situation. Related internal discussion: https://trello.com/c/frjiaRtk/42-group-discussion-how-to-get-better-at-giving-feedback
+<b>Micro-learning</b> I am a big fan of micro-learning or the learning that happens in really small chunks but in a regular manner. Shiva-s approach to <b>Python Question of the Day</b> is a good example of micro-learning. But you do not need anyone bright to guide your micro-learning. You can do that daily by simply asking yourself better questions as you go about your day. Just being curious about how something works or following up on a new word or an attempt to avoid a usual step or re-ordering your routine can keep your work fresh and while helping you learn something new.
```

Comment:
In the above point dont we need shiva's point. I can see hyphen over here

Reply:
@Annapooraniqxf2 - Good catch! This is a limitation of the Skype sender. It does not know how to deal with quotes. So I have avoided them.